community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/Workers.java

```diff
@@ -72,6 +72,18 @@ public Throwable await() throws InterruptedException
         return error;
     }
 
+    public Throwable awaitStrict()
```

Comment:
No, but it's a general purpose method and matches `awaitAndThrowOnError` which also has a `strict` counterpart. I'm not too worried about it being there.